autoria.py

```diff
@@ -0,0 +1,242 @@
+"""Python auto.ria.com API.
+
+Python implementation of API intended for calulating
+average used car prices that are sold on http://auto.ria.com
+"""
+
+
+import requests
+import json
+from fnmatch import fnmatch
+
+
+def make_request(url: str) -> json:
+    """Send get request and return data in JSON.
+
+    Args:
+        url - string with url returning needed
+        data (categories, bodystyles etc.)
```

Comment:
```
        url - resource URL to send the request too.
```

In this case the method is generic, it can return anything, even not from the autoria at all, so no need to mention categories, bodystyles, etc.